k9mail/src/main/java/com/fsck/k9/preferences/AccountSettings.java

```diff
@@ -224,6 +224,12 @@
         s.put("notifyContactsMailOnly", Settings.versions(
                 new V(42, new BooleanSetting(false))
         ));
+        s.put("openPgpHideSignOnly", Settings.versions(
```

Comment:
This seems to be missing the `openPgpProvider` setting.

Reply:
This is on purpose: Given the connection to some crypto provider that is saved in this setting, and that OpenKeychain requires some kind of setup for authentication, importing the openPgpProvider from a settings dump will likely cause more problems than it's worth, and the user will have to go to the dialog regardless.